packages/dx-react-demos/src/material-ui/localization/basic.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {
+  FilteringState,
+  LocalFiltering,
+  GroupingState,
+  LocalGrouping,
+  EditingState,
+  PagingState,
+  LocalPaging,
+} from '@devexpress/dx-react-grid';
+import {
+  Grid,
+  TableView,
+  TableHeaderRow,
+  TableFilterRow,
+  TableGroupRow,
+  GroupingPanel,
+  TableEditRow,
+  TableEditColumn,
+  PagingPanel,
+  DragDropContext,
+} from '@devexpress/dx-react-grid-material-ui';
+
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../demo-data/generator';
+
+const tableViewMessages = {
+  noData: 'Keine Daten verfügba',
+};
+const editColumnMessages = {
+  addCommand: 'Neue Zeile',
+  editCommand: 'Bearbeiten',
+  deleteCommand: 'Entfernen',
+  commitCommand: 'Speichern',
+  cancelCommand: 'Abbrechen',
+};
+const groupingPanelMessages = {
+  groupByColumn: 'Ziehen Sie eine Spalte hierhin, um danach zu gruppieren',
+};
+const filterRowMessages = {
+  filterPlaceholder: 'Filter...',
+};
+const pagingPanelMessages = {
+  showAll: 'Alle',
+  rowsPerPage: 'Zeilen pro Seite',
+  info: 'Zeile {from} von {to}({count} Elemente)',
```

Comment:
Translated back to English, this reads "Line {from} of {to}({count} Elements)". This seems weird - maybe it should be like "Line 10 to 15 (47 Elements)"? If so, the German should be "Zeilen {from} bis {to} ({count} Elemente)" - also note the added space before the opening paren.